vulture_os/gui/crontab/api_clients_parser.py

```diff
@@ -84,9 +92,13 @@ def api_clients_parser():
 
     processes = []
     for frontend in api_clients_parser:
-        p = Process(target=execute_parser, name=frontend.name, args=(frontend.to_dict(),))
-        p.start()
-        processes.append(p)
+        if node_selected(current_node, frontend):
+            p = Process(target=execute_parser, name=frontend.name, args=(frontend.to_dict(),))
+            p.start()
+            processes.append(p)
+
+    if not processes:
+        return
```

Comment:
seems superfluous, the case is handled properly in the while loop

Reply:
Deleted as suggested